cmd/user-agent/src/main.js

```diff
@@ -107,7 +109,9 @@ async function trustblocStartupOpts() {
         assetsPath: defaultTrustBlocStartupOpts['assetsPath'],
         blocDomain: ('blocDomain' in startupOpts) ? startupOpts['blocDomain'] : defaultTrustBlocStartupOpts['blocDomain'],
         walletMediatorURL: ('walletMediatorURL' in startupOpts) ? startupOpts['walletMediatorURL'] : defaultTrustBlocStartupOpts['walletMediatorURL'],
-        'log-level': ('log-level' in startupOpts) ? startupOpts['log-level'] : defaultTrustBlocStartupOpts['log-level']
+        'log-level': ('log-level' in startupOpts) ? startupOpts['log-level'] : defaultTrustBlocStartupOpts['log-level'],
+        'agent-username': ('agent-username' in startupOpts) ? startupOpts['agent-username'] : defaultTrustBlocStartupOpts['agent-username'],
```

Comment:
Here default opts for username is '', but in start command you made it mandatory.

Reply:
Removed the default opt.